benchmark/manifests/ModelServerService.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
```

Comment:
Could this whole file just be removed, and instead run`kubectl expose` ?

Reply:
Done, thanks for the feedback!